proto/translate.go

```diff
@@ -12,14 +12,19 @@ var (
 	timePtrType = reflect.TypeOf((*time.Time)(nil))
 	timeType    = timePtrType.Elem()
 	mapStrInf   = reflect.TypeOf((map[string]interface{})(nil))
+
+	epoch1970 = time.Date(1970, 1, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
The epoch1970 stuff is dealing with the fact that the protobuf zero ts is Jan 1 1970 where Go's time.Time has a zero value of Jan 1, 0. So we consider anything prior to jan 1 1970 to mean the same thing. In practice this wont be a problem.